### PR TITLE
Update homepage for short intro

### PR DESCRIPTION
**Pull request by jonhartzler**
The initiative team had interest in having the site intro and the introduction to the exhibit be 2 different things.   

I Changed the home page to display the short intro by a smaller image.  Then I moved the long intro modal to an Introduction section ahead of the sections.

### CODE REVIEW

**Comment by danthewolfe** [comment]
:+1:

File: app/assets/javascripts/components/collection/CollectionIntro.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    id: React.PropTypes.string,
+  },
+
+  style: function() {
+    if(this.props.height) {
+      return {
+        maxHeight: this.props.height + 'px',
+        overflow: 'hidden',
+      };
+      } else {
```
Comment (jaronkk) [inline app/assets/javascripts/components/collection/CollectionIntro.jsx]: This line and the closing brace after the return need to have one less level of indentation

File: app/assets/javascripts/components/collection/CollectionIntro.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      return {};
+      }
+  },
+
+  componentDidMount: function() {},
+
+  render: function() {
+    return (
+      <div style={this.style()} id={this.props.id}>
+        <p className="collection-intro" dangerouslySetInnerHTML={{__html: this.props.collection.short_description}} />
```
Comment (jaronkk) [inline app/assets/javascripts/components/collection/CollectionIntro.jsx]: This should be a div tag if we're inserting html into it.

File: app/assets/javascripts/components/modal/Info.jsx [hunk 3]
```diff
@@ -4,7 +4,16 @@ var React = require('react');
...
+    return ( <div>
+            <div className="about-collection">
+              <h2>About Collection</h2>
+              <h3>Name of Curator</h3>
+              <p><em>Title or Position</em><br/>Hesburgh Libraries</p>
+              <p>p: 124-123-1234<br/>e: name@nd.edu</p>
+            </div>
+      <p>Exhibits is a service of the <a href="http://library.nd.edu">Hesburgh Libraries of Notre Dame</a>.</p><p>For Technical Support call <a href="tel:5746316258">(574) 631-6258</a>  or email <a href="mailto:rfox2@nd.edu?subject=Support%20Request">rfox2@nd.edu</a>.</p></div>)
+
+    ;
```
Comment (jaronkk) [inline app/assets/javascripts/components/modal/Info.jsx]: Can you tidy up the indentation here, and make sure the semicolon is immediately after the closing parenthesis?
```jsx
    return (
      <div>
        <div className="about-collection">
          <h2>About Collection</h2>
          <h3>Name of Curator</h3>
          <p><em>Title or Position</em><br/>Hesburgh Libraries</p>
          <p>p: 124-123-1234<br/>e: name@nd.edu</p>
        </div>
        <p>Exhibits is a service of the <a href="http://library.nd.edu">Hesburgh Libraries of Notre Dame</a>.</p><p>For Technical Support call <a href="tel:5746316258">(574) 631-6258</a>  or email <a href="mailto:rfox2@nd.edu?subject=Support%20Request">rfox2@nd.edu</a>.</p>
      </div>
    );
```

**Comment by jaronkk** [comment]
Once my three code comments are addressed, I'm okay with this being merged.

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**